sdks/python/tox.ini

```diff
@@ -61,7 +61,8 @@ commands =
   pip --version
   {toxinidir}/scripts/run_tox_cleanup.sh
   python apache_beam/examples/complete/autocomplete_test.py
-  python setup.py nosetests
+  # Disable log capture due to BEAM-6603.
+  python setup.py nosetests --nologcapture
```

Comment:
I suggest --logging-level=WARNING here and below

Reply:
ms